### ISSUE TITLE
error starting batch file

### ISSUE THREAD

**Issue by graphixillusion** [gerardog/gsudo#6]
As the title says, with the just released 0.5.2 version i got this error when i try to start a batch file as admin:

```
Server Error:System.ComponentModel.Win32Exception (0x80004005): Unable to find specified file
   in System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   in gsudo.Helpers.ProcessFactory.StartInProcessAtached(String filename, String arguments)
   in gsudo.ProcessHosts.AttachedConsoleHost.<Start>d__0.MoveNext()
Info: Elevated process exited with code 999
```

**Comment by gerardog** [comment]
I was able to reproduce the issue. Thanks for reporting. I will add a test case for this scenario.
I think StartWithCreateProcess doesn't run batch files if the `.bat` extension is not specified.

I will release a fix. In the meantime, please use.

`gsudo batchfile.BAT` --> Add '.bat'
or 
`gsudo cmd /c batchfile`

**Comment by gerardog** [comment]
This was fixed in [v0.5.3](https://github.com/gerardog/gsudo/releases/tag/v0.5.3).
Please let me know if this fixed the issue
Thanks for reporting!

**Comment by graphixillusion** [comment]
I confirm that with the .3 update everything works good.
